_posts/2021-06-22-choosing-right-transformation.md

```diff
@@ -0,0 +1,114 @@
+
+## A Brief Refresher on Spheroids, Datums, and Projections
+
+To understand what's going on, let's go through a [brief refresher](https://desktop.arcgis.com/en/arcmap/10.3/guide-books/map-projections/about-the-geoid-ellipsoid-spheroid-and-datum-and-h.htm) on the geodetic foundations that projections are built upon. It all starts with a ___spheroid___, a simple mathematical approximation of the earth's shape: almost, but not quite, a sphere.
+
+The spheroid is used to create a ___datum___, which provides a more locally accurate model of the earth's surface. These take into account the "lumpiness" of the earth. A datum can be used as it's own "projection" (really, a geographic coordinate system) to define locations in latitude and longitude. Common datums include WGS84, NAD27, and NAD83.
+
+Side note: the fun part is that datums shift over time, so just saying "NAD83" is actually incomplete.  Because a datum can be tied to monuments on earth, it can shift over time due to tectonics. We mostly skip over this fact in GIS ([unless you live on the west coast](https://www.xyht.com/surveying/links-need-seeing-to-datum-epochs-and-how-to-understand-them/), apparently), but a complete definition of a datum also includes the epoch, or the date when it was updated. Unfortunately, most projections and GIS software gloss over this fact, which is another reason GIS means Get It Surveyed.
+
+Using the word properly, a ___projection___ spreads this 3D model of the surface of the earth onto a flat, two-dimensional plane. These are the projections we all know and love (or [hate](https://xkcd.com/977))—Robinson, Mercator, UTM, etc.
+
+## Irreconcilable Differences
+
+The WGS84 datum and the NAD83 datum started out identical. However, because WGS84 and NAD83 are tied and adjusted to different sets of points on the earth's surface (NAD83 focuses on the North American area, while WGS84 focuses on the entire world), they have drifted apart over time like a celebrity marriage.
+
+Because a projection using NAD83 and another using WGS84 are no longer starting from the same spot, any projected coordinates calculated in both no longer match. While the difference is slight on a planetary scale, it often works out to around a meter here in Utah.
+
+## Maybe Not So Irreconcilable
+
+Fortunately, we can use [___transformations___](https://desktop.arcgis.com/en/arcmap/10.3/guide-books/map-projections/choosing-an-appropriate-transformation.htm) to figure out the differences between datums. Each different transformation uses its own methods and starting reference points to translate coordinates.
+
+The trick is choosing the right transformation. The pitfall is that the default is usually wrong if you're going from NAD83 to WGS84.
```

Comment:
**thought**: i'm not sure it's really wrong. it's probably more right, but different than what we use to convert our data to wgs84.